### ISSUE TITLE
apollo-angular schematics has a fixed version of apollo-angular and @apollo/client packages

### ISSUE THREAD

**Issue by Naikno** [kamilkisiela/apollo-angular#2133]
**Describe the bug**

Into schematics the versions of apollo-angular and @apollo/client packages are fixed : 

```
 'apollo-angular': '^4.2.0',
 '@apollo/client': '^3.0.0',
```
**To Reproduce**

Try to 
```
ng add apollo-angular
```
or simply check on
https://github.com/kamilkisiela/apollo-angular/blob/master/packages/apollo-angular/schematics/install/index.ts

**Expected behavior**

The user can choose the versions, or the versions are updated to the latest

**Comment by PowerKiKi** [comment]
Indeed, it's not ideal. A PR that always use the latest version of packages would be most welcome.
